tests/annotated_binary/README.md

```diff
@@ -0,0 +1,92 @@
+# Annotated Flatbuffer Binary
+
+This directory demonstrates the ability of flatc to annotate binary flatbuffers 
+with helpful annotations. The resulting annotated flatbuffer binary (afb)
+contains all the binary data with line-by-line annotations.
+
+## Usage
+
+Given a `schema` in either plain-text (.fbs) or already compiled to a binary
+schema (.bfbs) and `binary` file(s) that was created by the `schema`.
+
+```
+flatc --annotate `schema` -- `binary`
+```
+
+### Example
+
+```
+cd tests\annotated_binary
+..\..\flatc --annotate annotated_binary.fbs -- annotated_binary.bin
+```
```

Comment:
perhaps we should run this automatically in `generate_code.py`?

Reply:
Yeah, I will add it to that once everything is settled.